content/en/blog/2024/hardening-the-collector-one.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Hardening the Collector Episode 1: A new default bind address'
+linkTitle: A new default bind address for the Collector
+date: 2024-07-02
+author: '[Pablo Baeyens](https://github.com/mx-psi) (OpenTelemetry, Datadog)'
+# prettier-ignore
+cSpell:ignore: awsfirehose awsproxy awsxray Baeyens jaegerremotesampling loki remotetap sapm signalfx skywalking splunk
+issue: 4760
+sig: Collector SIG
+---
+
+The OpenTelemetry Collector recently went through a security audit sponsored by
+the [CNCF](https://www.cncf.io/). As part of this process we recently published
```

Comment:
Can we use a similar wording from the CVE blog post? "sponsored by, facilitated by, executed by..."?

Reply:
Done, I used the same wording